accordion.js

```diff
@@ -162,13 +158,23 @@
 		allTitles.keydown(function(e) {
 			var code = e.which;
 
-			// 13 = Return, 32 = Space
+			// 13 = Return, 32 = Space, 27 = Esc
 			if ((code === 13) || (code === 32)) {
 				// Simulate click on title
 				$(this).click();
 			}
 		});
 
+		allTitles.on('keyup', function(e) {
+		var code = e.which;
+
+			if (code === 27) {
+				$(this).parent().next('.accordion-content').slideUp();
```

Comment:
There is a `closeItem(ele)` function you can call here, passing the accordion item title element, rather than recreating that functionality.

Reply:
Awesome, I'll substitute that in instead.